packages/ckeditor5-build-helpjuice/src/hjeditor.js

```diff
@@ -353,7 +353,7 @@ HelpjuiceEditor.defaultConfig = {
 			},
 			{
 				name: 'externalProvider',
-				url: /.*\.(ogg|mp4|webm)/i,
+				url: /.*\.(mpg|mp4|wmv|mpeg|webm|mkv|flv|vob|ogv|ogg|avi|rm|rmvb|asf|amv|mp2|mpe|mpv|m4v|svi|3gp|mov)/i,
```

Comment:
sometimes users might include a URL with some parameters, usually for analytical reasons, can we cover it in this regex?

Reply:
Yes. I tried to insert video URL with query parameters and it worked. You can try it on staging. I deployed the latest changes there.